build/rakelib/cluster.rake

```diff
@@ -119,6 +119,42 @@ namespace :cluster do
     end
   end
 
+  desc "Deploy ebrelayer"
+  namespace :ebrelayer do
+    task :deploy, [:chainnet, :provider, :namespace, :image, :image_tag, :ethereum_websocket_address, :ethereum_contract_address, :ethereum_private_key] do |t, args|
+      check_args(args)
+
+      cmd = %Q{helm upgrade sifnode ../build/helm/sifnode \
+        --set sifnode.env.chainnet=#{args[:chainnet]} \
+        --install -n #{ns(args)} \
+        --set image.tag=#{image_tag(args)} \
+        --set image.repository=#{image_repository(args)} \
```

Comment:
`image_repository` will pull in `sifchain/sifnoded` if arg is not set. This will need its own function to default to `"sifchain/ebrelayer"`